source/blog/_posts/2015-08-11-seo-engine.md

```diff
@@ -0,0 +1,242 @@
+---
+title: The Shopware SEO engine
+tags:
+    - SEO
+    - search engine optimization
+    - Shopware
+
+categories:
+- dev
+indexed: true
+github_link: blog/_posts/2015-08-11-seo-engine.md
+
+authors: [dn]
+---
+The term "SEO" summarizes various features and functionalities which target to increase the ranking of a given website
+within search engines. This will usually include "speaking URLs", rich snippets, meta tags and search engine optimized
```

Comment:
changed it to "semantic URLs" - in germany "speaking urls" is a thing, but probably the term is a germanism